src/langcheck/metrics/metric_inputs.py

```diff
@@ -378,8 +378,8 @@ def get_standard_metric_inputs(
     | tuple[IndividualInputType, IndividualInputType] = None,
     reference_outputs: IndividualInputType
     | tuple[IndividualInputType, IndividualInputType] = None,
-    additional_params: dict[str, IndividualInputType] | None = None,
-    additional_params_to_prompt_var_mapping: dict[str, str] | None = None,
+    additional_inputs: dict[str, IndividualInputType] | None = None,
+    additional_input_name_to_prompt_var_mapping: dict[str, str] | None = None,
     required_params: list[str],
 ) -> MetricInputs:
     """Create a metric inputs object with the standard parameters
```

Comment:
We can change this to

Create a metric inputs object with the standard parameters
(i.e. generated_outputs, prompts, sources, reference_outputs) and the
specified additional parameters.